css/src/components/steps.scss

```
@@ -0,0 +1,22 @@
+$steps-gap: 7px;
+$steps-size: 8px;
+$steps-size-double: 16px;
+
+.steps {
+	display: flex;
+	gap: $steps-gap;
+
+	.step {
+		width: $steps-size;
+		height: $steps-size;
+		border-radius: 50%;
+		background-color: $secondary;
+
+		&.step-selected {
+			width: $steps-size-double;
```

As a possibility:

```suggestion
			width: 2 * $steps-size;
```

This way we can use fewer variables, and if the step-size changes, the wider version automatically updates as well?